pkg/acsengine/artifacts.go

```diff
@@ -268,6 +274,19 @@ func kubernetesArtifactSettingsInitAgent(profile *api.Properties) []kubernetesFe
 	}
 }
 
+func getAddonString(input, destinationPath, destinationFile string) string {
```

Comment:
can this be unit tested?

Reply:
<img width="509" alt="screen shot 2018-11-13 at 12 16 36 pm" src="https://user-images.githubusercontent.com/2658224/48440580-17516900-e73e-11e8-89f2-e9cd491e8adf.png">

I've run tests and seen that the method is already covered

Reply:
great